cdk/app/lib/microservices.ts

```diff
@@ -27,6 +27,23 @@ export class Microservices extends cdk.Stack {
         gossipEncryptKey: serverProps.gossipKeySecret,
         tls: true,
         consulDatacenter: serverProps.serverDataCenter,
+        port: 3000,
+        consulChecks: [
+          {
+            checkid  : "server-http",
+            name     : "HTTP health check on port 3000",
+            http     : "http://localhost:3000/health",
+            method   : "GET",
+            timeout  : "10s",
+            interval : "2s",
+          }
+        ],
+        healthCheck: {
```

Comment:
consul health check and ECS health check can't be mixed in my test. should we add comments here to inform reader to choose only one health check?